hadoop-hdds/docs/content/security/SecurityAcls.md

```diff
@@ -172,3 +172,16 @@ $ ozone sh bucket removeacl -a user:testuser:r[DEFAULT] /vol1/bucket2
 ACL user:testuser:r[DEFAULT] removed successfully.
 ```
 
+## Differences Between Ozone ACL and S3 ACL
+
+Ozone ACLs and S3 ACLs differ primarily in their scope and support.
+
+- **S3 ACLs**: Currently, only S3 Bucket ACL is supported in Ozone. S3 Object ACL is not yet implemented. Any `PutObjectAcl` request will result in a `501: Not Implemented` response code.
```

Comment:
As mentioned in HDDS-11738, seems bucket ACL write operation doesn't work that well currently. So we can mark this as alpha / beta feature first. AFAIK, currently S3 bucket ACL has not really been used by any Ozone users.

Reply:
For `GetObjectAcl`, probably we also need to return '501: Not Implemented' as well. We can raise a ticket for this.

Reply:
Thanks @ivandika3 for pointing that out. I’ve created a [Jira ticket](https://issues.apache.org/jira/browse/HDDS-13056). Let me know if you have time to work on it. Otherwise, I’m happy to take it over.

Reply:
@chungen0126 Thanks for raising the ticket. Please feel free to work on it. I can review it afterwards.

Reply:
Ok. Updated. @chungen0126 if you plan to take on HDDS-13056, please update this doc too.